tests/integration/test_cross_stage_features.py

```diff
@@ -64,18 +64,10 @@ def test_submission_links(self):
         def _assert_upload(upload, no_upload, title, link, filename, uploaded_by):
```

Comment:
@e-kolpakov I think you can remove the `uploaded_by` parameter (function body does not reference it anymore).